src/core/settings.h

```diff
@@ -10,6 +10,7 @@ typedef struct QUIC_SETTINGS {
     BOOLEAN PacingDefault    : 1;
     BOOLEAN MigrationEnabled : 1;
     BOOLEAN DatagramReceiveEnabled  : 1;
+    BOOLEAN ServerResumeOrZeroRtt : 2;
```

Comment:
It's not a `BOOLEAN` if you use two bits. Make it a `uint8_t` then.

Reply:
I was trying to piggy-back off the already allocated `BOOLEAN` to use up more for the bits with less waste.